### PR TITLE
doc(gwk): polish English doc and upload Chinese doc

### PR DESCRIPTION
**Pull request by KyeGuo**
adjust docs, update content of code block, add zh

### CODE REVIEW

File: docs/cli_zh.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
 # Fling 的 CLI 使用
 
-TBD
+在这份文档中，我们将介绍 Fling 中命令行界面（CLI）的使用方式。
+
+## 版本
```
Comment (kxzxvbk) [inline docs/cli_zh.md]: Explain this clearer, such as "Check the version of Fling", also for English doc.

File: docs/cli_zh.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
...
+
+```shell
+fling run -c argzoo/mnist/mnist_fedper_cnn_toy_config.py -p personalized_model_pipeline -s 1,2,3
+```
+
+以下是关于每个参数的含义解释。
+
+- -c / --config：您想要使用的配置文件。
+- -p / --pipeline：您想要执行的 pipeline。此 pipeline 应包含在 `fling.pipeline` 中。
+- -s / --seed：您想要运行的 seeds。它可以是一个整数，也可以是类似示例中的多个整数。默认情况下，该 seed 被设定为0。
```
Comment (kxzxvbk) [inline docs/cli_zh.md]: Explain the ``seed`` here briefly. You can refer to: https://en.wikipedia.org/wiki/Random_seed. Also add this for English doc.

**Comment by kxzxvbk** [comment]
Make sure that if you modify the doc in code, please also modify the corresponding source code file.

File: docs/meaning_for_configurations_zh.md [hunk 3]
```diff
@@ -1,3 +1,245 @@
...
+
+请注意，以下配置文件即为默认配置文件。换句话说，如果用户定义的配置中某些键不存在，将默认使用以下文件中的值。
+
+以下是默认配置文件，您也可以在[这里](https://github.com/kxzxvbk/Fling/blob/main/argzoo/default_config.py)查看：
+
+```python
+import platform
+
+default_exp_args = dict(
+    # Configurations about data.
```
Comment (kxzxvbk) [inline docs/meaning_for_configurations_zh.md]: Translate comments in this file into Chinese.

File: docs/meaning_for_configurations_zh.md [hunk 4]
```diff
@@ -1,3 +1,245 @@
 # 配置文件各字段含义
 
-TBD
\ No newline at end of file
+## 一个示例
```
Comment (kxzxvbk) [inline docs/meaning_for_configurations_zh.md]: "一个实例" -> “默认配置实例”. Also for English doc.

File: docs/meaning_for_configurations_zh.md [hunk 5]
```diff
@@ -1,3 +1,245 @@
...
+        # The saved model checkpoint to start from. If it is set to ``None``, the training process
+        # will start from scratch.
+        resume_path=None,
+        # Whether to print config is the command line.
+        print_config=False,
+    ),
+)
+```
+
+## 其他教程
```
Comment (kxzxvbk) [inline docs/meaning_for_configurations_zh.md:133]: 其他 -> 其它

File: docs/meaning_for_configurations_zh.md [hunk 6]
```diff
@@ -1,3 +1,245 @@
...
+
+```python
+finetune_parameters=dict(
+    # This means to only fine-tune parameters except those whose names have keywords listed in `keywords`
+    name='except',
+    keywords=['fc', 'bn']
+)
+```
+
+`group.aggregation_parameters` 控制在联邦学习框架下应该聚合哪些参数，对于它的设置和上面所述的 `learn.finetune_parameters` 是类似的。
```
Comment (kxzxvbk) [inline docs/meaning_for_configurations_zh.md:174]: Please an example here. See FedPer config: https://github.com/kxzxvbk/Fling/blob/main/argzoo/cifar10/cifar10_fedper_resnet_config.py.
Explain concretely what part is personalized, what part is fine-tuned.

File: docs/meaning_for_configurations_zh.md [hunk 7]
```diff
@@ -1,3 +1,245 @@
...
+
+### launcher
+
+- 除了 "serial" 外，`launcher.name` 键还有其他用途。在除了 Linux 以外的操作系统上，默认配置 "serial" 意味着每个客户端上的所有操作都是串行执行的，这可能不够高效。我们可以使用多进程方法加速这个过程。
```
Comment (kxzxvbk) [inline docs/meaning_for_configurations_zh.md]: 用途 -> 模式。Also modify the English doc.

File: docs/meaning_for_configurations_zh.md [hunk 8]
```diff
@@ -1,3 +1,245 @@
...
+3) `num_proc` 指的是程序中使用的进程数。这个值的最优值可能会受到您执行的任务、所使用的硬件、神经网络、数据等的影响...... 您应该在您的系统上仔细调整这个值，以寻找最佳的选择。以下是在具有 4 核 CPU 的 A100 上，batch_size=32 的简单基准测试：
+
+| num_proc          | 1     | 2     | 3     | 4     | 8     |
+| ----------------- | ----- | ----- | ----- | ----- | ----- |
+| MNIST + CNN       | 21.7s | 14.9s | 15.3s | 16.5s | 25.2s |
+| CIFAR10 + ResNet8 | 45.2s | 32.1s | 30.1s | 32.5s | 40.4s |
+
+### data.transforms
+
+- `include_default` 键用于指定是否包括默认的数据增强方法。例如，如果您正在使用 CIFAR100 数据集并设置 `include_default=True` ，那么您使用的实际数据变换将是：
```
Comment (kxzxvbk) [inline docs/meaning_for_configurations_zh.md]: Add descriptions about default augmentaton and examples, such as: where the default augmentation of CIFAR100 is defined, what is the aug
